src/plugins/recordTypes/organization/forms/default.jsx

```diff
@@ -0,0 +1,82 @@
+const template = (pluginContext) => {
+  const {
+    React,
+  } = pluginContext.lib;
+
+  const {
+    Panel,
+    Row,
+    Cols,
+    Col,
+  } = pluginContext.layoutComponents;
+
+  const {
+    Field,
+    InputTable,
+    Subrecord,
+  } = pluginContext.recordComponents;
+
+  return (
+    <Field name="document">
+      <Panel name="info" collapsible>
+        <Field name="orgTermGroupList">
+          <Field name="orgTermGroup">
+            <Panel>
+              <Row>
+                <Field name="termDisplayName" />
+                <Field name="termType" />
+                <Field name="termStatus" />
+              </Row>
+
+              <Row>
+                <Field name="termLanguage" />
+                <Field name="termPrefForLang" />
```

Comment:
This looks weird to me. Try putting an empty col at the end of the row. That will make termLanguage shorter, and line up with termDisplayName.